ZhuoHeiChaUI/Events/InitializeGameStateEventArgs.cs

```diff
@@ -9,6 +9,6 @@ public class InitializeGameStateEventArgs : EventArgs
     {
         public List<int> cardBefore;
         public List<int> cardAfter;
-        public List<int> tributeList;
+        public List<int> PlayerTypeListThisRound;
```

Comment:
Let's be consistent and use public properties instead of public fields throughout the project when defining members of an `EventArgs` class.

Reply:
Done